lib/screens/register/register_screen.dart

```diff
@@ -104,118 +108,164 @@ class _RegisterFormState extends State<RegisterForm> {
+            SizedBox(height: spacing),
+            TextFormField(
+              controller: _passwordController,
+              validator: _validatePassword,
+              decoration: InputDecoration(
+                suffixIcon: IconButton(
+                  icon: Icon(_passwordVisible ? Icons.visibility : Icons.visibility_off),
+                  onPressed: _togglePasswordVisibility,
+                ),
+                labelText: "Enter password",
+                border: OutlineInputBorder(),
               ),
-              labelText: "Confirm password",
-              border: OutlineInputBorder(),
+              obscureText: !_passwordVisible,
             ),
-            obscureText: !_passwordVisible,
-          ),
-          Padding(
-            padding: EdgeInsets.all(8),
-            child: Text("Available to be a:"),
-          ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-            children: [
-              Row(
-                children: [
-                  Checkbox(
-                    value: _availableToMentor,
-                    onChanged: _toggleAvailableToMentor,
-                  ),
-                  Text("Mentor"),
-                ],
+            SizedBox(height: spacing),
+            TextFormField(
+              controller: _confirmPasswordController,
+              validator: _validatePassword,
+              decoration: InputDecoration(
+                suffixIcon: IconButton(
+                  icon: Icon(_passwordVisible ? Icons.visibility : Icons.visibility_off),
+                  onPressed: _togglePasswordVisibility,
+                ),
+                labelText: "Confirm password",
+                border: OutlineInputBorder(),
               ),
-              Row(
-                children: [
-                  Checkbox(
-                    value: _needsMentoring,
-                    onChanged: _toggleNeedsMentoring,
+              obscureText: !_passwordVisible,
+            ),
+            Padding(
+              padding: EdgeInsets.all(8),
+              child: Text("Available to be a:"),
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                Row(
+                  children: [
+                    Checkbox(
+                      value: _availableToMentor,
+                      onChanged: _toggleAvailableToMentor,
+                    ),
+                    Text("Mentor"),
+                  ],
+                ),
+                Row(
+                  children: [
+                    Checkbox(
+                      value: _needsMentoring,
+                      onChanged: _toggleNeedsMentoring,
+                    ),
+                    Text("Mentee"),
+                  ],
+                ),
+              ],
+            ),
+
+            Column (
+              children: [
+                CheckboxListTile(
```

Comment:
why is this in a column?

Reply:
Hey, @Techno-Disaster ! 
Because putting it in a row caused this checkbox to be in the same line as the two check boxes above it (**mentor**  and **mentee** checkboxes)

Reply:
Everything worked well after changing it from row to column.

Reply:
i think you put it inside the line 170 row? did you try putting it in the column on line 110?

Reply:
No no. 
I tried putting it as a row in the column on line 110. 
Doesn't work.

Reply:
It gives, 'BoxConstraints forces an infinite width.' error.

Reply:
I tried again just now, @Techno-Disaster. 
Putting it as a row inside the column on line 110 doesn't work.

Reply:
@MaheKhan no i mean just put the checkboxlisttile as a 110 column child. not column>row>child

Reply:
> @MaheKhan can you add a gif of the signup page in the PR body?

okay, @Techno-Disaster . I'll do that.

Reply:
> @MaheKhan no i mean just put the checkboxlisttile as a 110 column child. not column>row>child

Okay, tried doing that now. Doesn't have any effect on the output though.

Reply:
will get on my laptop later and check. kinda confusing on a mobile screen